<launch>
  <node pkg="wr_logic_ai" name="obstacle_avoidance" type="obstacle_avoidance.py" output="screen"/>
  <!-- <node pkg="wr_logic_ai" name="dead_reckoning" type="dead_reckoning.py" output="screen"/> -->
  <node pkg="wr_logic_ai" name="testData" type="inputFakeData.py" output="screen"/>
  
</launch>